node/src/accountant/db_big_integer/big_int_db_processor.rs

```diff
@@ -277,35 +280,56 @@ impl<'params, T: TableNameDAO> BigIntSqlConfig<'params, T> {
 }
 
 // This is a native Rusqlite format for data supply to their interface
-type RusqliteParamPair<'params> = (&'params str, &'params dyn ToSql);
-// This is a representation of the above, plus providing a way to display the values (not possible natively)
-type DisplayableRusqliteParamPair<'params> = (&'params str, &'params dyn DisplayableParamValue);
+// (<param_name_str>, <param_value_behind_reference>)
+pub struct RusqliteParamPair<'params> {
+    pub name: &'params str,
+    pub value: &'params dyn ToSql,
+}
 
-// To be able to display things that implement ToSql
-pub trait DisplayableParamValue: ToSql + Display {}
+type RusqliteNativePrimitivePair<'params> = (&'params str, &'params dyn ToSql);
```

Comment:
What does the first part of the tuple represent? Is it a data value? If it's not, can you please make it more intention revealing?